src/features/CellEditing.js

```diff
@@ -3,12 +3,39 @@
 var Feature = require('./Feature');
 var CellEditor = require('../cellEditors/CellEditor');
 
+var KEYS = {
+    RETURN: 'RETURN',
+    RETURNSHIFT: 'RETURNSHIFT',
+    DELETE: 'DELETE',
+    BACKSPACE: 'BACKSPACE',
+    SPACE: 'SPACE',
+    F2: 'F2'
+};
+
 /**
  * @constructor
  * @extends Feature
  */
 var CellEditing = Feature.extend('CellEditing', {
 
+    isVisibleChar: function(char, event) {
```

Comment:
1. Is it your intention to add to the `CellEditing` public interface? Because if not, we prefer local functions called with `.call(this,`.
2. making these four into functions is kind of pointless when called once, but OK, for sake of making these logic methods overridable

Reply:
No, there's no need to expose this (i've only created the functions to simplify handleKeyDown). I'll update this.